TrainingApp/Utils/AppHelper.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  AppHelper.swift
+//  TrainingApp
+//
+//  Created by Dagmawit Alemayehu on 24/01/2023.
+//
+
+import CoreData
```

Comment:
did you use CoreData? if not - remove it please

Reply:
Yes, I'm using it for 'NSManagedObject' which is inherited by the class.